big_tests/tests/mod_global_distrib_SUITE.erl

```diff
@@ -1139,3 +1165,30 @@ refresh_mappings(NodeName, Config) ->
 trigger_rebalance(NodeName, DestinationDomain) ->
     rpc(NodeName, mod_global_distrib_server_mgr, force_refresh, [DestinationDomain]),
     timer:sleep(1000).
+
+user_receives(User, Bodies) ->
+    ExpectedLength = length(Bodies),
+    Messages = escalus_client:wait_for_stanzas(User, ExpectedLength),
+    SortedMessages = order_by_seqnum(Messages),
+    case length(Messages) of
+        ExpectedLength ->
+            Checks = [escalus_pred:is_chat_message(Body, Stanza) || {Body, Stanza} <- lists:zip(Bodies, SortedMessages)],
+            case lists:all(fun(Check) -> Check end, Checks) of
+                true ->
+                    ok;
+                false ->
+                    ct:fail({user_receives_failed, {wanted, Bodies}, {received, SortedMessages}, {check, Checks}})
+            end;
+        _ ->
+            ct:fail({user_receives_not_enough, {wanted, Bodies}, {received, SortedMessages}})
+    end.
+
+refresh_hosts(Hosts) ->
+   [rpc:call(ct:get_config({hosts, Host, node}), mod_global_distrib_mapping_redis, refresh, []) || Host <- Hosts].
+
+connect_steps_with_sm() ->
```

Comment:
`_with_sm` might not specify if we want stream management or resumption. Maybe a more specific name, but I'm not stubborn about this.